CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# Release 2.13.0 08-07-2020
+
+- Update envoy to 7/6/2020 (#217)
```

Comment:
Only keep customer facing bugs/feature requests

- Create FilterStats in client_cache (#216)
- Fix bug in processing HTTP response, unit tests for client cache CHECK (#212)
- Support api config versioning:  add v6 to api folder name and package name (#210)